pkg/security/config/dump.go

```diff
@@ -107,15 +107,17 @@ var (
 	PROTOBUF StorageFormat = "protobuf"
 	// DOT is used to request the dot format
 	DOT StorageFormat = "dot"
-	// Profile is used to request the Secl profile format
+	// Profile is used to request the generation of a profile
 	Profile StorageFormat = "profile"
+	// SecL is used to request the Secl policy format
+	SecL StorageFormat = "secl"
 
 	strToFormats = make(map[string]StorageFormat)
 )
 
 // AllStorageFormats returns the list of supported formats
 func AllStorageFormats() []StorageFormat {
-	return []StorageFormat{JSON, PROTOBUF, DOT, Profile}
+	return []StorageFormat{JSON, PROTOBUF, DOT, Profile, SecL}
```

Comment:
Why do we have a mix of UPPER and Mixed case here ? Since you are touching this, could we sync on UPPER only ?

Reply:
Haha I actually prefer lower only, upper feels weird to me 😏